src/table/settings/game_settings.ini

```diff
@@ -134,6 +137,98 @@ str      = STR_CONFIG_SETTING_DISTANT_JOIN_STATIONS
 strhelp  = STR_CONFIG_SETTING_DISTANT_JOIN_STATIONS_HELPTEXT
 post_cb  = [](auto) { CloseWindowById(WC_SELECT_STATION, 0); }
 
+[SDT_VAR]
+var      = depot.rail_depot_types
+type     = SLE_UINT8
+from     = SLV_EXTENDED_DEPOTS
+flags    = SF_GUI_DROPDOWN
+def      = 1
+min      = 1
+max      = 3
+interval = 1
+str      = STR_CONFIG_SETTING_RAIL_DEPOT_TYPES
+strhelp  = STR_CONFIG_SETTING_RAIL_DEPOT_TYPES_HELPTEXT
+strval   = STR_CONFIG_SETTING_ONLY_STANDARD_DEPOT
+post_cb  = DepotSettingsChanged
+cat      = SC_EXPERT
+
+[SDT_VAR]
+var      = depot.road_depot_types
+type     = SLE_UINT8
+from     = SLV_EXTENDED_DEPOTS
+flags    = SF_GUI_DROPDOWN
+def      = 1
+min      = 1
+max      = 3
+interval = 1
+str      = STR_CONFIG_SETTING_ROAD_DEPOT_TYPES
+strhelp  = STR_CONFIG_SETTING_ROAD_DEPOT_TYPES_HELPTEXT
+strval   = STR_CONFIG_SETTING_ONLY_STANDARD_DEPOT
+post_cb  = DepotSettingsChanged
+cat      = SC_EXPERT
+
+[SDT_VAR]
+var      = depot.water_depot_types
+type     = SLE_UINT8
+from     = SLV_EXTENDED_DEPOTS
+flags    = SF_GUI_DROPDOWN
+def      = 1
+min      = 1
+max      = 3
+interval = 1
+str      = STR_CONFIG_SETTING_WATER_DEPOT_TYPES
+strhelp  = STR_CONFIG_SETTING_WATER_DEPOT_TYPES_HELPTEXT
+strval   = STR_CONFIG_SETTING_ONLY_STANDARD_DEPOT
+post_cb  = DepotSettingsChanged
+cat      = SC_EXPERT
+
+[SDT_BOOL]
+var      = depot.adjacent_depots
+from     = SLV_MULTITILE_DEPOTS
+def      = true
+cat      = SC_EXPERT
+
+[SDT_BOOL]
+var      = depot.distant_join_depots
+from     = SLV_MULTITILE_DEPOTS
+def      = true
+str      = STR_CONFIG_SETTING_DISTANT_JOIN_DEPOTS
+strhelp  = STR_CONFIG_SETTING_DISTANT_JOIN_DEPOTS_HELPTEXT
+post_cb  = [](auto) { CloseWindowByClass(WC_SELECT_DEPOT); }
+
+[SDT_VAR]
+var      = depot.depot_spread
+type     = SLE_UINT8
+from     = SLV_MULTITILE_DEPOTS
+def      = 1
```

Comment:
By defaulting to 1, this effectively disables this new feature you've spent so much time creating. Perhaps using the same default limit as stations (12) would be more appropriate?

Reply:
I also have my doubts about which should be the default value. The max depot spread before the feature was 1, so maybe it is better to set it as 1 as default.